### PR TITLE
pic cover and save pic

### PR DESCRIPTION
**Pull request by stephenkingsley**
1. click pic and show the cover
2. click the right button in the cover to save pic

### CODE REVIEW

**Comment by terkelg** [comment]
Nice! I'll look at it

**Comment by terkelg** [comment]
I haven't forgot you @stephenkingsley - I'm just too busy at the moment to make the code review.
I'll look at it asap

**Comment by stephenkingsley** [comment]
ok, thx ~

**Comment by stephenkingsley** [comment]
@terkelg may you review this pr as soon as possible, because I had some idea can not do it.

File: browser.js [hunk 1]
```diff
@@ -113,3 +116,109 @@ document.addEventListener('DOMContentLoaded', (event) => {
...
+        }
+        if (error) {
+          body = done;
+        }
+        const notificationOptions = {
+          title,
+          body,
+          icon: './static/icon.png'
+        };
+        const notification = new Notification(title, notificationOptions);
```
Comment (terkelg) [inline browser.js]: Notification is not defined, where do you get that?

File: browser.js [hunk 2]
```diff
@@ -1,5 +1,8 @@
 'use strict';
+const https = require('https');
+const fs = require('fs');
 const electron = require('electron');
+const { dialog } = require('electron').remote;
```
Comment (terkelg) [inline browser.js]: Please use the already defined ```electron``` variable from the line above *(line 4)*, or call it directly from ```electron.remote```

File: browser.css [hunk 3]
```diff
@@ -230,3 +230,32 @@ html.os-win32 .back-btn { top: 0; }
   border: none;
   margin: 0;
 }
+
+body {
```
Comment (terkelg) [inline browser.css]: This is not related to this commit

**Comment by brandonmcconnell** [comment]
I'd love to be able to save pictures as well! What do you mean by "show cover"? Some other great features would be managing multiple accounts, accessing managing messages, and viewing stories, though I don't know if Instagram's API allows any of that.

File: browser.js (hunk 1 above)
Comment (stephenkingsley) [inline browser.js]: `Notification` is inline in electron. [you can see this example](https://github.com/hokein/electron-sample-apps/blob/master/notifications/window.js#L16)

File: browser.css (hunk 3 above)
Comment (stephenkingsley) [inline browser.css]: let this app can drag. because when I scroll to the bottom, this app can not drag. you can try it.

**Comment by stephenkingsley** [comment]
@terkelg what do you think this pr? I modified `const { dialog } = electron.remote;`

**Comment by stephenkingsley** [comment]
@terkelg when I run `npm test`, the console show `Notification' is not defined.`  and I think this can ignore it?

**Comment by stephenkingsley** [comment]
@brandonmcconnell just like this

**Comment by stephenkingsley** [comment]
@brandonmcconnell just like this

**Comment by brandonmcconnell** [comment]
Oh nice, looks good! And are you guys working on desktop notifications as well? That would be sweet!

**Comment by brandonmcconnell** [comment]
Also, just an alternate icon I threw together in Photoshop and switched out for my copy. Figured I'd share.

2016_instagram_logo_new_s

**Comment by brandonmcconnell** [comment]
Also, just an alternate icon I threw together in Photoshop and switched out for my copy. Figured I'd share. It's styled to match the same spacing, sizing, and shadows from standard Mac apps.

2016_instagram_logo_new_s

**Comment by brandonmcconnell** [comment]
Also, just an alternate icon I threw together in Photoshop and switched out for my copy. Figured I'd share. It's styled to match the same spacing, sizing, and shadows from standard Mac apps.

2016_instagram_logo_new_s

And here's how it looks on the dock.

image

**Comment by stephenkingsley** [comment]
@brandonmcconnell notifications is working now! when save pic success, this app will show the notifications.

**Comment by brandonmcconnell** [comment]
Great! What kind of notifications do you mean?

I was saying it would be cool to get notifications as people like, comment, mention you, tag you, or when someone you have push notifications turned on for posts a photo.

**Comment by stephenkingsley** [comment]
@brandonmcconnell you can see #7 , I can do this. but I want to discuss some great solutions

**Comment by brandonmcconnell** [comment]
Gotcha! Well I'm excited to see whatever you come up with. Also here to brainstorm if you're interested in getting a feedback on concepts or ideas. Looks and works great!

**Comment by brandonmcconnell** [comment]
Any way to not only rename the app but have change the name that appears in the top Apple bar when open?

**Comment by terkelg** [comment]
@brandonmcconnell please add another issue for these things

**Comment by stephenkingsley** [comment]
this project is died? @terkelg

**Comment by anatolinicolae** [comment]
@stephenkingsley definitely not dead. @terkelg is busy for few days this week but we've been planning the project rewrite. Don't you worry about the cool things coming soon, as soon as me and Terkel get some free time to work on this.

**Comment by terkelg** [comment]
Closing for now. Too much happened to the code since this PR. Sorry, I was too busy back then to maintain the project.
